bookkeeper-server/src/test/java/org/apache/bookkeeper/client/GenericEnsemblePlacementPolicyTest.java

```diff
@@ -132,9 +132,9 @@ public void testReplaceBookie() throws Exception {
                     lh.addEntry(value);
                 }
             }
-            assertEquals(2, customMetadataOnNewEnsembleStack.size());
```

Comment:
I still don't understand why do you need to call 'legacy' method.  in 4.4.0, all the legacy methods are interface method (no default behavior), a customized implementation already implements them.